utils/config-schema.json

```diff
@@ -78,6 +78,15 @@
           "properties": {
             "tub": {
```

Comment:
`tub` should be optional (along with `vat`, `daiJoin`, `wethJoin`, `cdpManager`) as they are only used in `dev` if provided, to built the mock migration contract.